packages/opentelemetry-plugin-ioredis/test/assertionUtils.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  SpanKind,
+  Attributes,
+  Event,
+  Span,
+  Status,
+} from '@opentelemetry/types';
+import * as assert from 'assert';
+import { ReadableSpan } from '@opentelemetry/tracing';
+import {
+  hrTimeToMilliseconds,
+  hrTimeToMicroseconds,
+} from '@opentelemetry/core';
+
+export const assertSpan = (
```

Comment:
This is getting used by multiple plugins (grpc, redis and postgres), maybe later we should move this into common place.

Reply:
#617